site/app/templates/NotificationSettings.twig

```diff
@@ -42,6 +42,44 @@
                 {% endif %}
             </div>
 
+            <div class="heading">
+                <h2>Registration</h2>
+                <hr>
+            </div>
+            <div class="option row">
+                <div class="option-desc col-md-4">
+                    <span>
+                        <span class="option-title">New Self Registration</span>
+                        <br>
+                        <span class="option-alt">Alert me when a student self registers for this course</span>
+                    </span>
+                </div>
+                <div 
+                    class="option-input 
+                    col-md-3 
+                    notification-setting-input">
```

Comment:
While we don't have any code formatter for Twig code currently, I think something like this would be a bit more clear.  This is what most of the HTML formatters I've seen tend to do.  Apply below as well.

```suggestion
                <div 
                    class="option-input 
                    col-md-3 
                    notification-setting-input"
                >
```